pom.xml

```diff
@@ -151,10 +151,12 @@
         <guava.version>32.1.1-jre</guava.version>
         <hudi.version>0.14.1</hudi.version>
         <pagehelper.version>6.1.0</pagehelper.version>
+        <mysql-connector-j.version>8.1.0</mysql-connector-j.version>
```

Comment:
bump to latest version to fix CVE

Reply:
Thank you very much for your advice, I have already fixed it, and I also wish you a happy New Year.